distribution/lib/Standard/Base/0.0.0-dev/src/Data/Numbers.enso

```diff
@@ -1132,8 +1127,18 @@ round_max_long = 99999999999999
 round_min_long : Integer
 round_min_long = -99999999999999
 
+## PRIVATE
+   Restrict rounding decimal_places parameter.
 check_decimal_places : Integer -> Function
 check_decimal_places decimal_places ~action =
     if decimal_places >= round_min_decimal_places && decimal_places <= round_max_decimal_places then action else
         msg = "round: decimal_places must be between " + round_min_decimal_places.to_text + " and " + round_max_decimal_places.to_text + " (inclusive), but was " + decimal_places.to_text
         Error.throw (Illegal_Argument.Error msg)
+
+## PRIVATE
+   Restrict allowed range of input to rounding methods.
+check_round_input : Number -> Function
+check_round_input n ~action =
+    if n >= round_min_long && n <= round_max_long then action else
+        msg = "Error: Integer.round can only accept values between " + round_min_long.to_text + " and " + round_max_long.to_text + " (inclusive), but was " + n.to_text
```

Comment:
This error can now be raised on `Decimal.round` as well. We should ensure that the message is not misleading.

Maybe that could work for both?
```suggestion
        msg = "Error: `round` can only accept values between " + round_min_long.to_text + " and " + round_max_long.to_text + " (inclusive), but was " + n.to_text
```